.github/workflows/build.yml

```diff
@@ -114,6 +114,7 @@ jobs:
         # stm32f4
         - 'feather_stm32f405_express'
         - 'stm32f411ve_discovery'
+        - 'songhe_stm32f401c'
```

Comment:
I just take a quick look at songhe board, look like it is a re-brand of another more popular called stm32f401_blackpill https://stm32-base.org/boards/STM32F411CEU6-WeAct-Black-Pill-V2.0.html . Like stm32f103 bluepill, it is a generic dev board that is popular and is produced and sold across internet.